chapters/planning-data-work.tex

```diff
@@ -1,147 +1,457 @@
-%------------------------------------------------
+% ----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
-Preparation for data work begins long before you collect any data.
-In order to be prepared to work on the data you receive, 
-you need to know what you are getting into.
+Preparation for collaborative data work begins long before you collect any data,
+and involves planning both the software tools you will use yourself
+and the collaboration platforms and processes for your team.
+In order to be prepared to work on the data you receive with a group,
+you need to plan out the structure of your workflow in advance.
 This means knowing which data sets and output you need at the end of the process,
-how they will stay organized and linked,
-what different types and levels of data you'll handle,
-and whether the data will require special handling due to volume or privacy considerations.
+how they will stay organized, what types of data you'll handle,
+and whether the data will require special handling due to size or privacy considerations.
 Identifying these details creates a \textbf{data map} for your project,
```

Comment:
We are mentioning **data map** here but then never specifically explain or give an example of this.

Reply:
Made non-specific in 106a3c90ebd7c609e1fe91762173cb4463c46301

Reply:
Improved in 106a3c90ebd7c609e1fe91762173cb4463c46301

Reply:
ok, this is pragmatic compromise. Given that we do not have a good template for data maps yet, maybe lets go with this.

Reply:
ok, this is pragmatic compromise. Given that we do not have a good template for data maps yet, maybe lets go with this.